tests/python_package_test/test_dask.py

```diff
@@ -1224,6 +1224,51 @@ def test_training_succeeds_when_data_is_dataframe_and_label_is_column_array(
     client.close(timeout=CLIENT_CLOSE_TIMEOUT)
 
 
+@pytest.mark.parametrize('task', tasks)
+@pytest.mark.parametrize('output', data_output)
+@pytest.mark.parametrize('init_score', [0.25, 0.75])
```

Comment:
what is the purpose of testing two different values?

Reply:
I had that initially because I was comparing to the local model and kind of wanted to avoid having an init_score equal to whatever it would choose automatically based on the label. I guess that can go away now

Reply:
ok then yes please just set it to one value. It can be some random nonsense values like `0.12345`